### DISCUSSION TITLE
Name change suggestions

### DISCUSSION THREAD

**Discussion by spenserblack** [spenserblack/repofetch#219] [Feedback]
`repofetch` obviously implies that it has something to do with repos. But that's not really what it is, now. It's more of a "stat fetcher" manager, where each stat fetcher is a plugin in this project.

So this project
- makes it easier to write a stat fetcher by handling formatting, coloring, etc., and allowing you to just define the stats and ASCII art
- manages stat fetchers, by allowing the user enable and disable plugins, and use a dotenv file to set environment variables that the stat fetchers might use

The new project name should capture this. Or, at the very least, it shouldn't imply that this project is *only* for repos.

:warning: This project is still very new, without much usage. I won't change the name until there is an interest in authoring 3rd-party plugins available that *don't* report repository stats. In other words, this project still *is* a repository-stat fetcher, and I won't change the name until it's potentially *not.*

**Comment by theiereman** [comment]
I thought of "anemicfetch" in reference to that anti-pattern https://en.wikipedia.org/wiki/Anemic_domain_model

This would make sense as the base tool wouldn't do much if there weren't plugins

**Reply by spenserblack** [reply]
Oh, that's an interesting idea!
